qcodes/instrument_drivers/tektronix/Keithley_2400.py

```diff
@@ -77,11 +77,30 @@ def __init__(self, name, address, **kwargs):
                            label='Current integration time')
 
         self.add_parameter('resistance',
-                           get_cmd=':READ?',
+                           get_cmd=self._get_volt_and_curr,
                            get_parser=self._resistance_parser,
                            label='Resistance',
                            unit='Ohm')
 
+    def _get_volt_and_curr(self) -> str:
```

Comment:
Why is the function called “volt and curr “ while READ returns 4 values? Should the name of this function be more generic?